concrete/js/build/core/app/heartbeat.js

```diff
@@ -0,0 +1,28 @@
+/* jshint unused:vars, undef:true, browser:true, jquery:true */
+/* global CCM_DISPATCHER_FILENAME */
+
+;(function(global, $) {
+'use strict';
```

Comment:
Can you fix scope indentation in this file?

Reply:
Well, I'd prefer to keep the indentation like this: I consider the leading `;(function(global, $) {` as a *header* (it encloses all the remaining part of the file).

But just ping me if you really want to fix it...

Reply:
I'm okay with it given that it's not our main language but I do feel strongly about scope indentation